kotsadm/pkg/airgap/update.go

```diff
@@ -116,6 +125,10 @@ func UpdateAppFromAirgap(a *app.App, airgapBundle multipart.File) (finalError er
 		appNamespace = os.Getenv("KOTSADM_TARGET_NAMESPACE")
 	}
 
+	if err := task.SetTaskStatus("update-download", "Creating application...", "running"); err != nil {
```

Comment:
this should indicate creating a version not an application. maybe `Creating app version...` ?